### DISCUSSION TITLE
How to activate extension with proper URI?

### DISCUSSION THREAD

**Discussion by rfc1941** [coder/code-server#5114] [Q&A] [answered]
Dear developers,
As I am developing a extension for both desktop VSCode and code-server, I find that there are problems activating extension from URI.  
More specifically, on desktop, we could do `vscode://<your_uri>&<param1>=...` in browser and VSCode desktop would be triggered and extension starts. However, on code-server, what's the proper way to do the same? Suppose my domain for code-server is: `https://code.code-server.com`, what should we add to the URL?

Thanks a lot for helping!

**Comment by jsjoeio** [comment, marked as answer]
I don't think this is something we support unfortunately

**Reply by rfc1941** [reply]
What about not activating extensions, but simply pass some parameters to them?

**Reply by jsjoeio** [reply]
I don't think we support that either
